cmd/jaeger/internal/extension/jaegerstorage/converter/empty_test.go

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package converter
```

Comment:
typically we use `converter` for stateless model transformers. I would call this `factoryadapter`